hair_salon/data/ir_ui_view.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="appointment.calendar_event_view_gantt_booking_user" model="ir.ui.view">
+        <field name="arch" type="xml">
+            <gantt position="attributes">
+                <attribute name="default_group_by">user_id</attribute>
+            </gantt>
+        </field>
+        <field name="active" eval="True"/>
+        <field name="inherit_id" ref="appointment.calendar_event_view_gantt_booking_resource"/>
+        <field name="model">calendar.event</field>
+        <field name="name">calendar.event.view.gantt.meeting.user</field>
+        <field name="type">gantt</field>
+    </record>
+    <record id="appointment_type_customization" model="ir.ui.view">
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//form[1]/sheet[1]/group[1]/group[@name='left_details']/div[1]" position="after">
+                    <field name="x_break_start_appointment" string="Break Start" options="{&quot;human_readable&quot;:false}" widget="float_time"/>
+                    <field name="x_break_end_appointment" options="{&quot;human_readable&quot;:false}" widget="float_time"/>
+                </xpath>
+            </data>
+        </field>
+        <field name="active" eval="True"/>
+        <field name="inherit_id" ref="appointment.appointment_type_view_form"/>
+        <field name="mode">extension</field>
+        <field name="model">appointment.type</field>
+        <field name="name">appointment.type.form customization</field>
+        <field name="priority">200</field>
+        <field name="type">form</field>
+    </record>
+    <odoo>
+        <record id="calendar_event_view_form_gantt_booking_inherit" model="ir.ui.view">
+            <field name="name">calendar.event.view.form.gantt.booking.inherit</field>
+            <field name="model">calendar.event</field>
+            <field name="inherit_id" ref="appointment.calendar_event_view_form_gantt_booking"/>
+            <field name="active" eval="True"/>
+            <field name="arch" type="xml">
+                <data>
+                    <xpath expr="//form/sheet/group/group" position="after">
+                        <div class="o_form_group o_inline">
+                            <label for="x_linked_to" invisible="not x_linked_to" string="Linked to:" class="oe_inline me-2"/>
+                            <field name="x_linked_to" invisible="not x_linked_to" options="{'no_create': True}" class="oe_inline"/>
+                        </div>
+                    </xpath>
+                </data>
+            </field>
+        </record>
+    </odoo>    
+    <record id="calendar_event_parent_customization" model="ir.ui.view">
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//form[1]/sheet[1]/group[1]/group[1]/div[1]" position="after">
```

Comment:
Studio exports those very accurate working xpath, but those can easily break due to a change in standard. So we therefore try to shorten this as much as possible (by providing the name of the div, for example)